### PR TITLE
Better Brazilian Portuguese support

### PR DESCRIPTION
**Pull request by platbr**
I had already contributed for improve Brazilian Portuguese support of AmigosShare tracker.
Now this is to improve BJShare Tracker.
This code only adds "Brazilian" at title to support Sonarr/Radarr detect Brazilian Portuguese content.
I also changed AmigosShare a little.

### CODE REVIEW

**Comment by ilike2burnthing** [comment]
@garfield69 / @RoloSoze can you check AmigosShare?

@DarkSupremo / @christianfds can you check BJShare?

File: src/Jackett.Common/Indexers/BJShare.cs [hunk 1]
```diff
@@ -394,6 +394,9 @@ private async Task<List<ReleaseInfo>> ParseUserSearchAsync(TorznabQuery query)
                                              titleElements[2] + " " + titleElements[4] + " " + string.Join(
                                                  " ", titleElements.Skip(6));
 
+                        if (Regex.IsMatch(release.Description, "(Dual|[Nn]acional|[Dd]ublado)"))
```
Comment (DarkSupremo) [inline src/Jackett.Common/Indexers/BJShare.cs:397]: You would probably need to add it on ParseLast24HoursAsync method aswell, that is run when rss sync is called

Comment (platbr) [reply src/Jackett.Common/Indexers/BJShare.cs:397]: Ok, I will check it. tks.

Comment (platbr) [reply src/Jackett.Common/Indexers/BJShare.cs:397]: Done.

**Review by DarkSupremo — APPROVED** [review]
LGTM

**Review by garfield69 — APPROVED** [review]

**Comment by ilike2burnthing** [comment]
[v0.17.710](https://github.com/Jackett/Jackett/releases/tag/v0.17.710)

Thanks all.

**Pull request merged**